github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -168,15 +168,12 @@ periodics:
       - "/bin/sh"
       - "-c"
       - >
-        mkdir -p /tmp && cd /tmp &&
-        echo 'cat /etc/github/oauth' > /tmp/git-askpass-helper.sh &&
-        chmod +x /tmp/git-askpass-helper.sh &&
-        export GIT_ASKPASS=/tmp/git-askpass-helper.sh &&
-        git clone https://github.com/kubevirt/project-infra.git &&
-        cd project-infra &&
-        autoowners --dry-run=false --github-login=kubevirt-bot --org=kubevirt --repo=project-infra --assign=dhiller --target-dir=. --target-subdir=github/ci/prow/files --config-subdir=jobs --github-token-path=/etc/github/oauth &&
-        git commit --amend --signoff --no-edit &&
-        git push -f "https://kubevirt-bot@github.com/kubevirt-bot/project-infra.git" HEAD:autoowners
+        cd /tmp && echo 'cat /etc/github/oauth' > /tmp/git-askpass-helper.sh && chmod +x /tmp/git-askpass-helper.sh
+        && export GIT_ASKPASS=/tmp/git-askpass-helper.sh
+        && git clone https://github.com/kubevirt/project-infra.git && cd project-infra
```

Comment:
Can you switch this git checkout to an `extra_refs` entry? https://github.com/kubevirt/project-infra/pull/352/files#diff-2f36afec0e979dc7f77141ce5908bd0d75f8fc5b552d91e50673cfd23c850ef0R209 has then a good example on how to best use the askpass script.